gloo.rst

```diff
@@ -80,5 +80,5 @@ vispy.gloo.gl - low level GL API
 
 Vispy also exposes a (low level) functional GL API.
 
- `vispy.gloo.gl docs <gl.html>`_
+ :doc:`vispy.gloo.gl docs <gl>`
```

Comment:
I'm surprised this isn't `:mod:` but maybe it does the same thing

Reply:
I added a `gl.rst` so it needs to point to the document. I could have linked to the module I think if I wouldn't have wanted a custom title, but I followed the pattern the gloo documentation does by putting it in a separate rst and autoloading the docs from the python source.